### DISCUSSION TITLE
What is REQUEST_INSTALL_PACKAGES being used for?

### DISCUSSION THREAD

**Discussion by matchboxbananasynergy** [z-huang/music#545] [Q&A] [answered]
Hey there! I was looking at the app's manifest and was wondering what this permission is for:

https://github.com/z-huang/InnerTune/blob/dev/app/src/main/AndroidManifest.xml#L9

**Comment by z-huang** [comment, marked as answer]
In previous versions, the app had a built-in updater, which needs REQUEST_INSTALL_PACKAGES permission. But now the built-in updater is removed, and I forgot to remove this permission.😅

**Reply by matchboxbananasynergy** [reply]
I see! Appreciate the quick reply. :) I'm assuming that since it's not needed, it'll be removed in the future (principle of least privilege and all)
